src/select/src/components/__tests__/Creatable.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import 'jest-styled-components';
+import Creatable from '../Creatable';
+
+const OPTIONS = [
+  { label: '0', value: 'zero' },
+  { label: '1', value: 'one' },
+  { label: '2', value: 'two' },
+  { label: '3', value: 'three' },
+  { label: '4', value: 'four' },
+];
+
+describe('Creatable', () => {
```

Comment:
I prefer having test files be in the same location as the component being tested, i.e.:

```
src/select/src/components/...
  |-- Creatable.js
  |-- Creatable.test.js
  |-- Select.js
  |-- Select.test.js
```